programs/client/Client.cpp

```diff
@@ -923,6 +923,7 @@ void Client::printHelpMessage(const OptionsDescription & options_description)
     std::cout << options_description.main_description.value() << "\n";
     std::cout << options_description.external_description.value() << "\n";
     std::cout << options_description.hosts_and_ports_description.value() << "\n";
+    std::cout << "All settings are documented at https://clickhouse.com/docs/en/operations/settings/settings and readable from system.settings.\n\n";
```

Comment:
Sorry, the grammar of my proposal was trash (lol), `[...] and available in system.settings` sounds better.

But to be honest, I would omit the hint to `system.settings`. It kind of duplicates the web documentation and in 99.9% of all cases all settings will take their default anyways (in the other 0.1% of all cases, the user sets them explicitly via CLI or config file but even then one would not need to look into system.settings for that).

Instead, you could perhaps link https://clickhouse.com/docs/en/operations/system-tables/settings from https://clickhouse.com/docs/en/operations/settings/settings (e.g .`All settings on this page are displayed in system.settings`)?

Another problem is that this the sentence reads a bit out-of-place with `--help` (which does not print settings). It actually only makes sense with `--help --verbose`. Is it possible to print it only in the latter case?

Reply:
So, if I understood correctly, we need to remove the hint for `system.settings`. Add a link to `system.settings` docs for https://clickhouse.com/docs/en/operations/settings/settings and try to show the hint when the user uses only the verbose version?

Reply:
yes